scripts/conformance_suites/run_efm_conformance_suite.py

```diff
@@ -2,20 +2,20 @@
 
 from arelle.CntlrCmdLine import parseAndRun
 
-
+# from https://www.sec.gov/structureddata/osdinteractivedatatestsuite
```

Comment:
Random Line for commentary:

It'd be kind of nice to just delete all of these `run_x_conformance_suite` files and just combine them with the test files. Lots of duplication going on